src/components/WorkoutCard/WorkoutCard.spec.js

```diff
@@ -1,5 +1,7 @@
 import WorkoutCard from './WorkoutCard'
 import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Button from '../Button'
```

Comment:
I wouldn't import the Button here as you then only test the Button and not the WorkoutCard. You should find the button in your WorkoutCard component instead.

Reply:
Thanks! I added the changes